### PR TITLE
IP addresses control

### PR DESCRIPTION
**Pull request by palayusuf**

### CODE REVIEW

File: src/test/java/org/example/IPAddressValidatorTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    @Test
+    public void testValidIPAddresses() {
+        assertTrue(validator.isValidIPAddress("192.168.1.1"));
+        assertTrue(validator.isValidIPAddress("0.0.0.0"));
+        assertTrue(validator.isValidIPAddress("255.255.255.255"));
+        assertTrue(validator.isValidIPAddress("127.0.0.1"));
+    }
+
+    @Test
+    public void testInvalidIPAddresses() {
```
Comment (palaevren) [inline src/test/java/org/example/IPAddressValidatorTest.java:19]: Can you add tests with numbers out of range?

**Review by palaevren — APPROVED** [review]
